examples/rust/tcp-echo.rs

```diff
@@ -271,22 +271,28 @@ impl Application {
     /// Runs the target echo server.
     pub fn run_server(&mut self) -> ! {
         let start: Instant = Instant::now();
+        let mut nclients: usize = 0;
```

Comment:
Since you also have the `clients` `Vec`, you wouldn't need this, as you could use `clients.len()`, but this is fine.